### PR TITLE
travis: enable gosimple

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

**Comment by jrick** [comment]
did gosimple remove that unused function or did you? it's fine but i'd add a comment saying "while here also remove..." in the commit message.  Pretty sure gosimple wouldn't do that since it is exported and could be imported elsewhere.

**Comment by dajohi** [comment]
gosimple sent me there for another reason.  Then I spotted the tickets variable was unused.  I'll just remove the whole function...

File: goclean.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@
 set -ex
 
 # Automatic checks
-test -z "$(gometalinter --disable-all --enable=gofmt $(glide novendor) | tee /dev/stderr)"
+test -z "$(gometalinter --disable-all --enable=gofmt --enable gosimple --deadline=4m $(glide novendor) | tee /dev/stderr)"
```
Comment (jrick) [inline goclean.sh]: why the deadline?

Comment (dajohi) [inline goclean.sh]: because gosimple is slow

Comment (jrick) [inline goclean.sh]: also it would be good to be consistent with using = vs not for the flags.

Comment (jrick) [inline goclean.sh]: if it is that slow i'm not so sure we actually want it running every time on the CI infrastructure.

Comment (jrick) [inline goclean.sh]: so i'm ok with this after all since it errors if it takes longer than the deadline.  it's not a silent error.

**Pull request merged**